tests/rbd/rbd_failback.py

```diff
@@ -0,0 +1,85 @@
+import time
```

Comment:
/rbd_failback.py - the name doesn't quite describe the action being performed in this module. You are basically trying to configure mirroring on newly created cluster, promoting images on new cluster to primary and demoting ones in old cluster right? 
Maybe some name like reconfigure_primary_cluster would be more suitable?